pkg/server/event_handler.go

```diff
@@ -80,8 +80,7 @@ const (
 
 // An interface for sending pmap event downstream.
 type Messenger interface {
-	Send(context.Context, *v1alpha1.PmapEvent) error
-	Cleanup() error
+	Send(context.Context, []byte, map[string]string) error
```

Comment:
Since we removed cleanup for messenger, we should do the same, which is when creating the messenger, instead of creating the clients inside the messenger [here](https://github.com/abcxyz/pmap/blob/bca64a43b358d27f9613407a2858bc788eacf88a/pkg/server/pubsub.go#L35), pass it directly.